bot/controllers/invite.js

```diff
@@ -59,20 +67,20 @@ class inviteController extends BaseController {
           if (err) {
             message.reply(errorMsg);
             util.log('Email not sent', err, 3);
-            message.author.send('Invite not sent ');
+            message.author.send('Invite not sent.');
           } else {
             util.log('Email details', info, 3);
             // for every invite sent the user will receive 1 point
-            getUserPointsandUpdate(message.author.id, 1);
+            getUserPointsandUpdate(message.author.id, invitePointsAwarded);
             // notify the user that the message has been sent
-            message.author.send('you have sent the invite');
+            message.author.send('The invite has been sent succesfully.');
           }
         });
       } else {
-        message.author.send('Sorry the invite could not be sent please contact a admin for assistance');
+        message.author.send('Sorry the invite could not be sent please contact a admin for assistance.');
       }
     });
-    return 'You have received 1 point';
+    return `Hey ${message.author}, you've received ${invitePointsAwarded} point(s) for sending this invite.`;
```

Comment:
Hey just a heads up, this fires no matter what. Even if there is an error in the invite process, it still tells them they earned points.